core/storage/db.py

```diff
@@ -0,0 +1,50 @@
+import peewee
+
+db = peewee.SqliteDatabase('outdoorfusion.db')
```

Comment:
can you maybe make a class in the pkg module that starts the db connection? So we can access it from the main function. 
basically, make a file in `pkg/storage/database.py` and just make a class that starts the peewee sqlite db